guide/additional-info/rest-api.md

```diff
@@ -4,128 +4,155 @@ REST APIs are extremely popular on the web and allow you to freely grab a site's
 
-Since we're only going to be making [GET](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) requests in this tutorial, we're going to use [Snekfetch.get](https://snekfetch.js.org/?api=snekfetch#Snekfetch.get). To require Snekfetch's GET request method, we'll do:
+To require snekfetch you'd do:
 
 ```js
-const { get } = require('snekfetch');
+const snekfetch = require('snekfetch');
 ```
 
-## Random Cat
+### Skeleton Code
 
-Random cat's API is available at [random.cat/meow](https://random.cat/meow) and returns a [JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON) response. To actually fetch the API, you're going to do the following:
+To start off, you're just going to be using this skeleton code:
 
 ```js
-get('https://random.cat/meow');
+const Discord = require('discord.js');
+
+const client = new Discord.Client();
+const prefix = '!';
+
+client.on('ready', () => {
+	console.log('Ready!');
+});
+
+client.on('message', async message => {
+	if (!message.content.startsWith(prefix) || message.author.bot) return;
+
+	const args = message.content.slice(prefix.length).split(/ +/);
+	const command = args.shift().toLowerCase();
+});
+
+client.login('pleaseinsertyourtokenheresothistutorialcanwork');
+```
+
+### Random Cat
+
+Random cat's API is available at https://random.cat/meow and returns a [JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON) response. To actually fetch data from the API, you're going to do the following:
+
+```js
+snekfetch.get('https://random.cat/meow');
 ```
 
 Now, of course it seems like this does nothing but what it's doing is launching a request to the random.cat server and random.cat is returning some JSON that contains a `file` property which is a string containing a link to a random cat. So, let's implement that into a command. The code should look similar to this:
 
 ```js
-const { body } = await get('https://random.cat/meow');
+if (command === 'cat') {
+	const { body } = await snekfetch.get('https://random.cat/meow');
 
-message.channel.send(body.file);
+	message.channel.send(body.file);
+}
 ```
 
 So, here's what's happening in this code:
-1) We're sending a GET request to `https://random.cat/meow`.
-2) random.cat sees our request and gets a random file from their database.
-3) random.cat then sends that file's URL as a JSON object that contains a link to the image/gif.
-4) Snekfetch's body property deserializes the JSON response using [JSON.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse].
-5) We then send the object's `file` property in Discord.
 
-<p class="warn">The response will only be parsed if the server's `Content-Type` header includes `application/json`. In some cases, you may have to get the `text` property instead of the `body` property and JSON.parse it yourself.</p>
+1) You're sending a `GET` request to random.cat.
```

Comment:
Still broken; needs to use `.` instead of `)`.
![broken list](https://i.imgur.com/AxlyC1k.png)